train.py

```diff
@@ -59,9 +83,9 @@
     checkpoint = ModelCheckpoint(checkpoint_file % args.prefix, monitor='val_loss')
     callbacks.insert(checkpoint)
 
-model.fit(x_train, y_train, batch_size=args.batch_size, nb_epoch=args.epochs,
-          validation_data=(x_val, y_val), callbacks=callbacks)
+model.fit([diff_train, comment_train], y_train, batch_size=args.batch_size, nb_epoch=args.epochs,
+          validation_data=([diff_val, comment_val], y_val), callbacks=callbacks)
 
-score, acc = model.evaluate(x_val, y_val, batch_size=args.batch_size)
+score, acc = model.evaluate([diff_val, comment_val], y_val, batch_size=args.batch_size)
```

Comment:
I think model.evaluate will return an array instead of scalars, to work with the multiple outputs, but I'm not sure about that yet.